perf-tool/src/client.cpp

```diff
@@ -162,20 +187,67 @@ void Client::closeClient()
     close(socketFd);
     printf("Closed connection with server.\n");
 }
+void printUsage()
+{
+    printf(" --portNo<p>: Required Port number\n --log<l> Output file name\n --host<h> host\n --usage<u> show help\n");
```

Comment:
The help is misleading. There should be a space or an = sign between the option name and the value.

Reply:
@mpirvu - space will not work for optional arguments. How about help as shown below?
```
./a.out -u
 --portNo<p>=Port
 --log<l>=Output_filename
 --host<h>=host
 --usage<u> shows usage

Eg:./a.out --portNo=port --log=logname --host=localhost
./a.out --p=port --l=logname --h=localhost
```

Reply:
It's fine to use `=` in the help topic. Using space worked in my little program.

Reply:
Actually I find that `--portNo<p>` confusing. It's either `--portNo=1234` or `-p 1234`, so let's just use the long options in the help

Reply:
Even for short options, we need to use `=` separator for `optional arguments`. New usage prints as below
```
Usage: ./a.out [OPTIONS]

Options:
--p, --portNo=Port          Specify port number to connect to server
--l, --log=logname          Specify Output file name, default log is clientLogs.json
--h, --host=hostname        Specify host, default is localhost
--u, --usage                shows usage

Eg: ./a.out --portNo=9002 --log=log.json --host=localhost
(OR) Use Short_Options: ./a.out --p=9002 --l=log.json --h=localhost
```